groovy-android-gradle-plugin/src/test/groovy/groovyx/functional/KotlinSupportSpec.groovy

```diff
@@ -17,17 +17,20 @@ package groovyx.functional
 
 import groovyx.functional.internal.AndroidFunctionalSpec
 
+import java.lang.Void as Should
```

Comment:
Are there any non-standard characters on this line? Or is it just the way GitHub renders spaces in import statements? I am seeing two red squares...

<img width="262" alt="screen shot 2017-11-17 at 11 37 11 am" src="https://user-images.githubusercontent.com/743291/32928177-b0c3963c-cb8b-11e7-907f-976b96907440.png">

Reply:
Those are just spaces. GitHub for some reason does that when using import aliasing.